README.md

```diff
@@ -70,14 +71,12 @@ Hey there! Welcome to Alveus.gg! There's a few ways that you can help contribute
 
 1. Install Node.js (see `engines` in `package.json` for the required versions), or use `fnm`/`nvm` to install the correct version of Node.js, and use `corepack enable` to use PNPM.
 2. Install dependencies: `pnpm install --frozen-lockfile`
-3. Create a [PlanetScale](https://planetscale.com/) account (free) or provide your own MySQL server, that should give you two DSN for the main and shadow database (something like `mysql://user:pass@us-east.connect.psdb.cloud/alveusgg?sslaccept=strict` and `mysql://user:pass@us-east.connect.psdb.cloud/alveusgg/shadow?sslaccept=strict`)
+3. Run `docker-compose up -d` from within `apps/website` to start a local MySQL database, and an S3 bucket with MinIO.
```

Comment:
Any reason we recommend the older `docker-compose` over `docker compose`?

Reply:
Habit lol, will swap